srw/srw.cpp

```diff
@@ -479,6 +541,7 @@ class srw {
 					}
 
 					stream->write((const char *)sph, total_size(sph) + sizeof(struct sph));
+					stream->close();
```

Comment:
Why we didn't it before? What has changed from worker's and client perspective with this change?